models/Application.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose")
+const Schema = mongoose.Schema
+
+const ApplicationSchema = new Schema({
```

Comment:
I'm wondering if we should rename this "Registration" - I know it only shows up behind the scenes, but I think "Registration" would make more sense for after a student is accepted. I think we can still have the statuses of 'new', 'accept', 'deny', or 'disabled' though.

Another thought - maybe you could add an `applicationRequired` field for the `Program` model - some of our programs do require applications and students need to be accepted, but other programs anyone can just sign up.